alcs-frontend/src/app/features/application/decision/decision.component.html

```diff
@@ -24,14 +24,14 @@ <h3>Decision</h3>
       <div>
         <strong>Modified&nbsp;By:&nbsp;</strong>
         <ng-container *ngFor="let modification of decision.modifiedBy">
```

Comment:
Do you still need this for loop? I dont see any usage of modification.

Reply:
done